modules/API/API.js

```diff
@@ -253,6 +254,9 @@ function initCommands() {
             sendAnalytics(createApiEvent('screen.sharing.toggled'));
             toggleScreenSharing(options.enable);
         },
+        'toggle-subtitles': () => {
```

Comment:
Let's add a boolean `enabled` parameter please. We started following that pattern because otherwise it makes it hard to use from the app side, since they need to keep the state.

Reply:
Added the setSubtitles command which has the enabled parameter. I kept the toggleSubtitles command also, I saw this pattern at setTileview and toggleTileview.